src/components/AssetManagementPanel/AssetManagementForm/AssetManagementForm.tsx

```diff
@@ -32,24 +36,34 @@ export const AssetManagementForm = ({
   onConnectToWallet,
   beneficiary,
   holder,
+  approvedBeneficiary,
+  approvedHolder,
   onSetFormAction,
   surrenderingState,
   onSurrender,
   onTransferHolder,
   holderTransferringState,
   onEndorseBeneficiary,
   beneficiaryEndorseState,
+  onApproveNewTransferTargets,
+  approveNewTransferTargetsState,
 }: AssetManagementFormProps) => {
   const isHolder = account === holder;
   const isBeneficiary = account === beneficiary;
   const canSurrender = isBeneficiary && isHolder;
   const canEndorseBeneficiary = isBeneficiary && isHolder;
+  const canNominateBeneficiaryHolder =
+    isBeneficiary &&
+    !isHolder &&
+    approvedBeneficiary === "0x0000000000000000000000000000000000000000" &&
+    approvedHolder === "0x0000000000000000000000000000000000000000";
```

Comment:
Dont need any of these conditions, the beneficiary can nominate new ones any. For instance if the holder do not agree to the values, he can ask the bene to re-nominate.